.travis.yml

```diff
@@ -10,9 +10,12 @@ env:
   - secure: HedklAPTdQAEhwsLhCwGco62/1Hq00nYgB+0MZYaVEb9KVdKxgZUswNidtzXVLha9TTOe3ewTmVjsd+Hs0wyyq6TZh8TUKlVPrJc1wd4wkurCK8JZjqUxBV2EydYmGM7j/CHpci1rcGv3QNQRwf2ZwfAoW5YPObJqqZvwk3GYgg=
   - secure: g+rMdXQTBwVb5S58Mj2uKbhwLE1N+Rc/3qGIEzfLY8V5Jj2beak8uNH+m/7dmRPBYLaL6QP5ndt6cY47w3lcXWGa00xeve12BHk9i5dvzx4hDZ18PoWGA5iB/U1Aj4I8USrASliyb2dqHtQg0V2nixgHuNdlJiwDgRNROHAdsQg=
 install: /bin/true
-script: ./ci/script/build.sh
-after_script:
-- ./ci/script/deploy.sh
+script:
+  - export BRANCH_NAME=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo $TRAVIS_BRANCH; else echo $TRAVIS_PULL_REQUEST_BRANCH; fi)
+#  - ./ci/script/build.sh
+  - ./ci/script/run_integrated_tests.sh
+after_success:
```

Comment:
I mean, I can still run both. The thing is, right now, the test project builds the link and the SDK, so I just wanted to save time and prevent it to build the link twice. I guess I can set it back, and add to a next PR a mechanism to use the built stuff from the build.sh script, if that makes sense.